examples/templates/incus/README.md

```diff
@@ -1,7 +1,7 @@
 ---
 display_name: Incus System Container with Docker
 description: Develop in an Incus System Container with Docker using incus
-icon: /icon/lxc.svg
+icon: ../../../site/static/icon/lxc.svg
```

Comment:
😅

Reply:
To be fair, both forms work. But all our other `examples/templates` use this relative form, and it's handy for confirming that the exact path exists.

Reply:
Yep, +1 for enforced consistency!